example/remap/remap.cc

```diff
@@ -272,7 +272,7 @@ TSRemapDoRemap(void *ih, TSHttpTxn rh, TSRemapRequestInfo *rri)
 
   // How to store plugin private arguments inside Traffic Server request processing block.
   if (TSHttpArgIndexReserve("remap_example", "Example remap plugin", &arg_index) == TS_SUCCESS) {
-    TSDebug(PLUGIN_NAME, "Save processing counter %lu inside request processing block\n", _processing_counter);
+    TSDebug(PLUGIN_NAME, "Save processing counter %" PRIu64 " inside request processing block\n", _processing_counter);
```

Comment:
I think this needs to be `%zu`. I ran into this issue before in another part of the code.